### PR TITLE
Fixes after the refactoring.

### PR DESCRIPTION
**Pull request by monicasarbu**
* The Topbeat.Config() method was not called
* Topbeat config is under Input
* Other changes: removed outdated comment, extra space in yml file

### CODE REVIEW

**Comment by monicasarbu** [comment]
@ruflin , can you please have a look at my changes?

File: main.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ func main() {
 	b.CommandLineSetup()
 
 	b.LoadConfig()
+	tb.Config(b)
```
Comment (ruflin) [inline main.go]: I missed that one somehow. Unfortunately it wasn't catched by the tests. I should change that.

File: topbeat.go [hunk 2]
```diff
@@ -21,7 +22,7 @@ type Topbeat struct {
 	procs        []string
 	procsMap     ProcsMap
 	lastCpuTimes *CpuTimes
-	TbConfig     TopConfig
+	TbConfig     ConfigSettings
```
Comment (ruflin) [inline topbeat.go]: As ConfigSettings only contains Input, can't we just skip it? https://github.com/elastic/topbeat/blob/master/config.go#L10

I would suggest to completely remove line 10-14

File: topbeat.go [hunk 3]
```diff
@@ -65,8 +66,17 @@ func (t *Topbeat) Run(b *beat.Beat) error {
 		time.Sleep(t.period)
 
 		err = t.exportSystemStats()
+		if err != nil {
```
Comment (ruflin) [inline topbeat.go]: We should probably do the same to packetbeat to check if all errors are catched and handled.

File: topbeat.go (hunk 2 above)
Comment (monicasarbu) [inline topbeat.go]: I don't think we can skip ConfigSettings. We need the Input field in order to have the section input in the config file.

Comment (ruflin) [inline topbeat.go]: Ah, yes. Forgot about this one. But we could remove line 14?

**Comment by ruflin** [comment]
Looks good to me. Should I merge?

**Pull request merged**